redux/middlewares/form.js

```diff
@@ -0,0 +1,6 @@
+export default () => next => action => {
+  if (typeof window !== 'undefined' && action.meta && action.meta.setErrors && action.error) {
```

Comment:
- `typeof window !== 'undefined'` was added only for `ReactGA`, please omit
- lets pass all formik actions to meta:
```js
  if (error && meta && meta.formikActions) {
    const { setErrors } = meta.formikActions;
    setErrors(payload);
  }
```